.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Think you can delete this file and the .vscode folder, and add .vscode to the .gitignore

Reply:
cool. done.